project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=1.5.7
```

Comment:
I would suggest `1.6.1`

Reply:
Great idea @mchv! I have now updated to 1.6.1.